src/components/elements/text/text.tsx

```diff
@@ -52,14 +79,26 @@ export const Text = (props: IGenericElementProps<ITextSchema>) => {
 	};
 
 	return (
-		<Element
-			id={id}
-			data-testid={getTestId(id)}
-			{...otherSchema}
-			// NOTE: Parent text body should be transformed into <div> to prevent validateDOMNesting error
-			{...(hasNestedFields() && { as: "div" })}
-		>
-			<Sanitize id={id}>{renderText()}</Sanitize>
-		</Element>
+		<>
+			<Element
+				id={id}
+				ref={contentDivRef}
+				maxLines={maxLines > 0 && !isMore ? maxLines : undefined}
```

Comment:
why do we need to check if maxLines > 0?

Reply:
just in case the value of maxLines to be 0 or negative eventually, then force it to undefined

Reply:
ok, you are right. expanded hook settled already, maxLines > 0 is not a must, removed it.